webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -69,7 +80,7 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
   });
   const [sortString, setSortString] = useState<string>('');
   const [experiments, setExperiments] = useState<Loadable<ExperimentWithTrial>[]>(
-    Array(page * PAGE_SIZE).fill(NotLoaded),
+    Array((page + 1) * PAGE_SIZE).fill(NotLoaded),
```

Comment:
q: what is the `+ 1` doing here?

Reply:
Since scroll view starts page at 0 while paged view starts page at 1, initially I wanted to somehow unify them and added the `+1`. I don't think we need it now.